spec/util_spec.rb

```diff
@@ -21,6 +21,15 @@
       end
     end
 
+    context "when there is an invalid byte sequence in the output" do
+      let(:output) { "branch-\xC3" }
+      let(:err_output) { "" }
+
+      it "replace the invalid byte sequence" do
```

Comment:
Is this still a correct description of this test? It seems we are now retaining, rather than replacing, the invalid byte sequence.

Reply:
Also, minor point, but this should read "replaces" (or "retains") rather than "replace".